introduction.md

```diff
@@ -1,2 +1,11 @@
-<h1> Hello I am a super human untent onm destroying the universe whahahahahahah</h1>
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?